crates/miden-lib/src/account/interface/mod.rs

```diff
@@ -0,0 +1,312 @@
+use alloc::{collections::BTreeSet, sync::Arc, vec::Vec};
+
+use miden_objects::{
+    account::{Account, AccountId, AccountProcedureInfo},
+    assembly::mast::{MastForest, MastNode, MastNodeId},
+    crypto::dsa::rpo_falcon512,
+    note::{Note, NoteScript},
+    Digest,
+};
+
+use crate::{
+    account::components::{
+        basic_fungible_faucet_library, basic_wallet_library, rpo_falcon_512_library,
+    },
+    note::scripts::{p2id, p2idr, swap},
```

Comment:
I wonder, and maybe this is a bad idea, if we should have constants with MAST roots of well-known interface procedures/scripts. These constants could be generated at build-time and then we'd be able to use them here without the need to look things up in libraries.

Reply:
I think `p2id().hash()` basically just looks up a single mast node in the forest and retrieves its hash, without having to compute it. Is it worth adding a separate constant for this?